cpp/ql/src/semmle/code/cpp/controlflow/ControlFlowGraph.qll

```diff
@@ -94,22 +94,6 @@ import ControlFlowGraphPublic
  */
 class ControlFlowNodeBase extends ElementBase, @cfgnode { }
 
-/**
- * Holds when `n2` is a control-flow node such that the control-flow
- * edge `(n1, n2)` may be taken when `n1` is an expression that is true.
- */
-predicate truecond_base(ControlFlowNodeBase n1, ControlFlowNodeBase n2) {
-  qlCFGTrueSuccessor(n1, n2)
-}
-
-/**
- * Holds when `n2` is a control-flow node such that the control-flow
- * edge `(n1, n2)` may be taken when `n1` is an expression that is false.
- */
-predicate falsecond_base(ControlFlowNodeBase n1, ControlFlowNodeBase n2) {
-  qlCFGFalseSuccessor(n1, n2)
-}
```

Comment:
These predicates are user-visible, so they'll have to go through a deprecation cycle: mark them as `deprecated` and start the qldoc with `DEPRECATED: Use ControlFlowNode.getA{False,True}Successor instead.`.

Reply:
Good point; done.